sphinx/source/bokeh/footer.html

```diff
@@ -42,6 +42,9 @@ <h6>Hands On</h6>
             <li class="nav-item">
               <a class="nav-link" href="https://demo.bokeh.org">Demos</a>
             </li>
+	    <li class="nav-item">
```

Comment:
Besides the issue with tabs, we can also fix the alignment here to be consistent with the previous tags.

Reply:
Made the changes, I must have messed up when I tried fixing this spacing, thanks again @pavithraes